snorkel/labeling/model/label_model.py

```diff
@@ -825,6 +857,7 @@ def fit(
         >>> label_model.fit(L)
         >>> label_model.fit(L, Y_dev=Y_dev)
         >>> label_model.fit(L, class_balance=[0.7, 0.3])
+        >>> label_model.fit(L, n_epochs=200, lr=0.05, l2=0.4, seed=2020)
```

Comment:
Let's add one of these keyword arguments to one of the above calls as a demonstration instead so that this doctest isn't too complex.